### DISCUSSION TITLE
Project Future....

### DISCUSSION THREAD

**Discussion by i0x404** [rafaelvcaetano/melonDS-android#935] [Q&A]
Hello,
I am wondering about the future of this emulator, is it going to be updated or is the project is just dead?

Because melonDS already updated their emulator to version 0.9.4 and this port is stuck on 0.9.3 and it hasn't been updated over almost 8 months or so.

Anyway really great project and I hope there is plans for updating and continuing to fix it. thanks a lot.

**Comment by YerekZero** [comment]
As stated by the dev in [this comment](https://github.com/rafaelvcaetano/melonDS-android/discussions/909#discussioncomment-2733566)  on a similar discussion on May 11th 2022,  life stuff put it on hold, but he was hoping to start development again soon. That is the last I have seen. 

Edit : Gammar, because I cannot type.

**Comment by FrenchPrick** [comment]
Hasn't this Question already been answered, the developer is just busy he
didn't abandon the project, but he might if you keep asking this stupid
questions.

**Reply by i0x404** [reply]
> Hasn't this Question already been answered, the developer is just busy he didn't abandon the project, but he might if you keep asking this stupid questions.

You're responding to a question that has 3 months since I posted about it and yes I got my answer but I can't delete the discussion, so in the next time check the date first before you comment.
